autogpt/config/ai_config.py

```diff
@@ -131,7 +133,7 @@ def construct_full_prompt(
             prompt_generator = plugin.post_prompt(prompt_generator)
 
         # Construct full prompt
-        full_prompt = f"You are {prompt_generator.name}, {prompt_generator.role}\n{prompt_start}\n\nGOALS:\n\n"
+        full_prompt = f"You are {prompt_generator.name}, {prompt_generator.role}\nThe OS you are running on is {os_name} {os_version}\n{prompt_start}\n\nGOALS:\n\n"
```

Comment:
Have you considered only printing this if `execute_local_commands` is enabled?

Reply:
@Pwuts That's a good idea, just added it into the latest commit.